src/compat/eme/custom_media_keys/is_custom_media_keys.ts

```diff
@@ -0,0 +1,23 @@
+/**
+ * Copyright 2015 CANAL+ Group
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+import { ICustomMediaKeys } from "./types";
+
+export function isCustomMediaKeys(
+  mediaKeys: MediaKeys|ICustomMediaKeys
+): mediaKeys is ICustomMediaKeys {
+  return (mediaKeys as ICustomMediaKeys)._setVideo !== undefined;
```

Comment:
The more I look at this, the more there is something that bothers me. This code makes no sense as itself, without the corresponding type information.

I think we may want to define a `ICustomMediaKeys` in the same file.

Reply:
After consideration, maybe it shouldn't even be needed at all now